bot/cogs/cyber.py

```diff
@@ -78,11 +82,25 @@ def __init__(self, bot: Bot):
         Gets information about a specific CyberStart Game level and challenge.
         If the date is before the start date of game (15th January 2019) it will redirect to game() instead
         """
-
         if datetime.date.today() < datetime.date(2019, 1, 15):
             await self.game.callback(self, ctx)
             return
 
```

Comment:
Two blank lines in a row will make Travis fail. When merging the two copies of `on_message`, don't forget to remove this stray newline to ensure Travis passes!